homeassistant/components/sensor/sensehat.py

```diff
@@ -0,0 +1,139 @@
+"""
+Support for Sense HAT sensors.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/
+"""
+import os
+import logging
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['https://github.com/RPi-Distro/python-sense-hat'
+                '/archive/master.zip#sense-hat==2.2.0']
```

Comment:
This module is available at [PyPI](https://pypi.python.org/pypi/sense-hat/2.2.0).

Can be changed to:

```python
REQUIREMENTS = ['sense-hat==2.2.0']
```